specs/agents/tracing-instrumentation-db.md

```diff
@@ -69,6 +69,10 @@ The following fields are relevant for database and datastore spans. Where possib
 | __**service.target._**__ |<hr/>|<hr/>|
 |`_.type`| `dynamodb` ||
 |`_.name`| e.g. `us-east-1` | Use same value as `context.db.instance` |
+| __**otel.attributes._**__ |<hr/>|<hr/>|
+TODO
+|`_["aws.dynamodb.table_names"]`| `[ MyTable ]` | The table names, if available. See [OTel Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.23.0/specification/trace/semantic_conventions/instrumentation/aws-sdk.md). Note: this must be a single dotted string key in the `otel.attributes` mapping -- for example `{"otel": {"attributes": {"aws.dynamodb.table_names": "[ MyTable ]"}}}` -- and *not* a nested object. |
```

Comment:
Should the example be `{"otel": {"attributes": {"aws.dynamodb.table_names": [ MyTable ]}}}` instead? I don't know if the quote should be around the brackets [] as it's supposed to be an array of strings.

Reply:
Good catch @estolfo! You're right, I'm going to change it

Reply:
Done!